pkg/quarterdeck/api/v1/api.go

```diff
@@ -32,6 +31,9 @@ type QuarterdeckClient interface {
 
 	// Project Resource
 	ProjectCreate(context.Context, *Project) (*Project, error)
+
+	// Users Resource
+	UserUpdate(context.Context, *User) error
```

Comment:
Do you think that `UserUpdate` should return a `*User` to be in line with the other API endpoints?